doc/user-guide/dask.rst

```diff
@@ -2,195 +2,151 @@
 
 .. _dask:
 
-Parallel computing with Dask
+Parallel Computing with Dask
 ============================
 
-Xarray integrates with `Dask <https://dask.org/>`__ to support parallel
-computations and streaming computation on datasets that don't fit into memory.
-Currently, Dask is an entirely optional feature for xarray. However, the
-benefits of using Dask are sufficiently strong that Dask may become a required
-dependency in a future version of xarray.
+`Dask <https://dask.org/?utm_source=xarray-docs>`__, a general purpose library for parallel computing, integrates with Xarray to handle larger-than-memory computations.
 
-For a full example of how to use xarray's Dask integration, read the
-`blog post introducing xarray and Dask`_. More up-to-date examples
-may be found at the `Pangeo project's gallery <http://gallery.pangeo.io/>`_
-and at the `Dask examples website <https://examples.dask.org/xarray.html>`_.
+If you’ve been using Xarray to read in large datasets or split up data across a number of files, you may already be using Dask:
 
-.. _blog post introducing xarray and Dask: https://stephanhoyer.com/2015/06/11/xray-dask-out-of-core-labeled-arrays/
+.. code-block:: python
+
+    import xarray
+
+    ds = xr.open_zarr("/path/to/data.zarr")
+    timeseries = ds["temp"].mean(dim=["x", "y"]).compute()  # Compute result
+
+Using Dask with Xarray feels similar to working with NumPy arrays, but on much larger datasets. The Dask integration is transparent, so you don’t need to manage the parallelism directly; Xarray and Dask handle these aspects behind the scenes. This makes it easy to write code that scales from small, in-memory datasets on a single machine to large datasets that are distributed across a cluster, with minimal code changes.
 
-What is a Dask array?
----------------------
+Examples
+--------
+
+- `Zonal averaging with the NOAA National Water Model <https://docs.coiled.io/user_guide/xarray.html?utm_source=xarray-docs>`_
+- `CMIP6 Precipitation Frequency Analysis <https://gallery.pangeo.io/repos/pangeo-gallery/cmip6/precip_frequency_change.html>`_
+- `Using Dask + Cloud Optimized GeoTIFFs <https://gallery.pangeo.io/repos/pangeo-data/landsat-8-tutorial-gallery/landsat8.html#Dask-Chunks-and-Cloud-Optimized-Geotiffs>`_
+- `Xarray + Dask Tutorial <https://tutorial.xarray.dev/intermediate/xarray_and_dask.html>`_
+
+
+Using Dask with Xarray
+----------------------
 
-.. image:: ../_static/dask_array.png
-   :width: 40 %
+.. image:: https://docs.dask.org/en/stable/_images/dask-array.svg
```

Comment:
It would probably be better to ship this image as part of the xarray repo, so that the link cannot become broken?